python/ray/tune/integration/wandb.py

```diff
@@ -326,6 +325,8 @@ def setup(self):
             os.path.expanduser(self.api_key_path) if self.api_key_path else None
         )
         _set_api_key(self.api_key_file, self.api_key)
+        wandb.require("service")
+        wandb.setup(settings=wandb.Settings(start_method="spawn"))
```

Comment:
```suggestion
        wandb.setup(settings=wandb.Settings(start_method=os.environ.get("WANDB_START_METHOD", "spawn")))
```

Maybe this so users can configure the behavior?

Reply:
I set this to spawn because, the last error that was coming up, was saying start_method fork is not supported, which later got fixed when I updated the requirements.txt file. Ideally, we don't even need to specify the start_method this way, as wandb automatically reads the `WANDB_START_METHOD` variable while invoking `setup()`. I'll remove this start method and check if the tests are still passing. Thanks for pointing it out.